DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -294,6 +294,43 @@ void Executor::operator()(const Model::PrintCommand& command)
     }
 }
 
+void Executor::operator()(const Model::WriteFileCommand& command)
+{
+    PIXScopedEvent(m_device->GetCommandList(), PIX_COLOR(255,255,0), "WriteFile: %s", command.resourceName.c_str());
+
+    try
+    {
+        auto resource = m_resources[command.resourceName];
+        auto fileData = m_device->Download(resource.Get());
+        auto& resourceDesc = m_model.GetResource(command.resourceName);
+        auto& bufferDesc = std::get<Model::BufferDesc>(resourceDesc.value);
+
+        std::ofstream file(command.targetPath.c_str(), std::ifstream::trunc | std::ifstream::binary);
+        if (!file.is_open())
+        {
+            throw std::ios::failure("Could not open file");
+        }
+
+        // TODO: Find a suitable folder so the parsing code can access ReadNpy(),
```

Comment:
Still thinking about this, given the resources don't include their dimensions 🤔. I suppose I'll add a `dimensions` field to the `writeFile` command (ideally they could be associated with the resource, but oh well).

Reply:
Ok, I decided to repeat the `dimensions` attribute inside the `writeFile` command.